tensorflow/contrib/cmake/tf_tools.cmake

```diff
@@ -73,7 +73,7 @@ add_executable(${transform_graph}
     $<TARGET_OBJECTS:tf_core_direct_session>
     $<TARGET_OBJECTS:tf_tools_transform_graph_lib>
     $<TARGET_OBJECTS:tf_core_kernels>
-    $<$<BOOL:${tensorflow_ENABLE_GPU}>:$<TARGET_OBJECTS:tf_core_kernels_cpu_only>>
+    $<$<BOOL:${tensorflow_ENABLE_GPU}>:$<$<BOOL:${BOOL_WIN32}>:$<TARGET_OBJECTS:tf_core_kernels_cpu_only>>>
```

Comment:
I'm not sure why this tool (and the subsequent ones) depend on the kernels at all. It seems like this might be making the executables *much* larger than they need to be! 

@petewarden Do any of these tools need to run kernels? (I think I've confirmed the `summarize_graph` doesn't, but I'm not sure about the others.)

Reply:
The transform_graph tool does require ops to be linked in, since it can run transform rules like fold_constants that actually run parts of the graph.

Reply:
Thanks for clarifying, Pete! Perhaps we can remove the kernels altogether from `summarize_graph` and `compare_graphs` then?